symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/AppAuthProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.symphony.bdk.app.spring.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Configuration Properties for enabling Application Authentication APIs.
+ */
+@Getter
+@Setter
+public class AppAuthProperties {
```

Comment:
Do we import `spring-boot-configuration-processor` as depdency (cf. https://www.baeldung.com/spring-boot-configuration-metadata#nested-properties)?

Reply:
Yes we have it [here](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/285/files#diff-fe5571c0f96e23877232698ad99f3361df1f1f0d3217b73a87db1b0c57307304R32)